docs/functions/cypress/pageobjects/FPaginatorPageObject.md

```diff
@@ -0,0 +1,180 @@
+---
+name: FPaginatorPageObject
+title: "FPaginatorPageObject"
+layout: article
+---
+
+Använd `FPaginatorPageObject` för att hämta information relaterat till {@link FPaginator paginator}.
+
+## currentPageButton()
```

Comment:
Jag tror det hade hjälpt med lite grafik här faktiskt, hade vi kunnat lägga in en `FPagionator` och på något vis highlightat vilken som är "current page button"?

```css
.current-page-button {
  outline: 2px dashed hotpink;
  outline-offset: 3px;
}

(har inte kollat vad klassen heter, men du förstår poängen)